File: xsolla-store-sdk/src/main/java/com/xsolla/android/store/entity/response/items/fields/Consumable.java
package com.xsolla.android.store.entity.response.items.fields;

import com.google.gson.annotations.SerializedName;

class Consumable {

    @SerializedName("usages_count")
    private int usagesCount;

    public int getUsagesCount() {
        return usagesCount;
    }
}

File: xsolla-store-sdk/src/main/java/com/xsolla/android/store/entity/response/common/Consumable.kt
package com.xsolla.android.store.entity.response.common

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize


@Parcelize
data class Consumable(
    @SerializedName("usages_count")
    val usagesCount: Long? = null
) : Parcelable